SCClassLibrary/Common/Math/Complex.sc

```diff
@@ -129,6 +137,9 @@ Complex : Number {
 	round { arg aNumber = 1.0;
 		^Complex(real.round(aNumber), imag.round(aNumber))
 	}
+	isInfinite {
```

Comment:
Not sure this method makes sense here.. we don't have a general is `isInfinite`, and it's difficult to tell from the name alone whether it should return true if one part or both parts are infinite. How would we resolve this with `isNaN`? Wouldn't `Complex(inf, nan)` be considered `NaN` first, and by definition be neither finite or infinite? Also just from a practical perspective, this will reduce performance.